docs/references/javascript/organization/members.mdx

```diff
@@ -1,42 +1,105 @@
 ---
-title: Organization Members
-description: These are all methods on the Organization class that allow you to manage the members of an organization.
+title: Organization membership methods
+description: Learn about the methods on the Organization object that allow you to manage the members of an organization.
 ---
 
-# Organization Members
+# Organization membership methods
 
-These are all methods on the [Organization](/docs/references/javascript/organization/organization) class that allow you to manage the members of an organization.
+These methods on the [`Organization`](/docs/references/javascript/organization/organization) object allow you to manage the memberships of an organization.
 
 ## `getMemberships()`
 
+Retrieves the list of memberships for the currently active organization.
+
 ```typescript
-function getMemberships(params?: GetMembersParams): Promise<PaginatedResponse<OrganizationMembership>>;
+function getMemberships(params?: GetMembersParams): Promise<ClerkPaginatedResponse<OrganizationMembership>>;
 ```
 
-Retrieve the members of the currently active organization.
-
 ### `GetMembersParams`
 
 | Name | Type | Description |
 | --- | --- | --- |
 | `initialPage?` | `number` | A number that can be used to skip the first n-1 pages. For example, if `initialPage` is set to 10, it is will skip the first 9 pages and will fetch the 10th page. |
 | `pageSize?` | `number` | A number that indicates the maximum number of results that should be returned for a specific page. |
-| `role` | `string` | The [roles](/docs/organizations/roles-permissions) of memberships that will be included in the response. |
+| `role?` | <code>[`OrganizationCustomRoleKey`](#organization-custom-role-key)[]</code> | The [roles](/docs/organizations/roles-permissions) of memberships that will be included in the response. |
```

Comment:
```suggestion
| `role?` | <code>[OrganizationCustomRoleKey](#organization-custom-role-key)[]</code> | The [roles](/docs/organizations/roles-permissions) of memberships that will be included in the response. |
```
The backticks made it nested in two `<code>` blocks:

![image](https://github.com/clerk/clerk-docs/assets/5156915/629bb7cd-d8a0-4581-a3e8-902eb747c52b)

Which makes me wonder, why do we need to use `<code>` here? Would the backticks (`) not work?

Reply:
The backticks would only cover `OrganizationCustomRoleKey` and its leave its array signature `[]` out of the snippet. And we cannot use backticks for the entire thing because of the MDX link - it just doesn't work. So we use <code>

Reply:
Good catch!